hera_sim/interpolators.py

```diff
@@ -360,3 +361,33 @@ def __init__(self, datafile, **interp_kwargs):
         super().__init__(datafile, **interp_kwargs)
         self._obj = "bandpass"
         self._check_format()
+
+
+class Reflection(FreqInterpolator):
+    """Complex reflection coefficient interpolator."""
+
+    def __init__(self, datafile, **interp_kwargs):
+        if "interpolator" not in interp_kwargs:
+            interp_kwargs["interpolator"] = "interp1d"
+        super().__init__(datafile, **interp_kwargs)
+        self._obj = "reflection"
+        self._check_format()
```

Comment:
Why isn't this called in the `__init__()` of the base class?

Reply:
It should be, and I can make that change. Somewhere down the line I want to rewrite this entire module, though, since I didn't fully understand the scope of the things we'd typically want to interface with when I wrote this module.

Reply:
Story of our life 😆